com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs

```diff
@@ -636,7 +636,12 @@ enum ClusterDepthSource : int
         int m_CurrentShadowSortedSunLightIndex = -1;
         HDAdditionalLightData m_CurrentSunLightAdditionalLightData;
         DirectionalLightData m_CurrentSunLightDirectionalLightData;
-        internal Light GetCurrentSunLight() { return m_CurrentSunLight; }
+
+        /// <summary>
+        /// Main directional Light for the HD Render Pipeline.
+        /// </summary>
+        /// <returns>The main directional Light.</returns>
+        public Light GetMainLight() { return m_CurrentSunLight; }
```

Comment:
Guess is weird to expose this Method inside the LightLoop class. Guess it should be in another location, maybe HDRenderPipeline. What are your thought?

Reply:
There isn't actually any lightloop class, it's just the name of the file. The function is inside the class HDRenderPipeline, which is `partial`, so defined in multiple files.

For user it will be accessible this way:
```cs
var sunLight = (RenderPipelineManager.currentPipeline as HDRenderPipeline).GetMainLight();
```

Reply:
ah right :)